p/themes/Origine/origine.css

```diff
@@ -2,10 +2,19 @@
 
 /*=== FONTS */
 @font-face {
-	font-family: "OpenSans";
-	src: url("../fonts/openSans.woff") format("woff");
+  font-family: 'Open Sans';
+  font-style: normal;
+  font-weight: 400;
+  src: url('../fonts/opensans.eot'); /* IE9 Compat Modes */
+  src: local('Open Sans'), local('OpenSans'),
+       url('../fonts/opensans.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
+       url('../fonts/opensans.woff2') format('woff2'), /* Super Modern Browsers */
+       url('../fonts/opensans.woff') format('woff'), /* Modern Browsers */
```

Comment:
Watch out for the casing: there is already a file `openSans.woff` with a big S. I suggest keeping the same casing, `openSans.xxx` (otherwise it must be changed everywhere).